guides/guides-base/src/components/SEO/SEO.jsx

```diff
@@ -1,6 +1,7 @@
 import React, { Component } from "react";
 import Helmet from "react-helmet";
 import urljoin from "url-join";
+import config from "../../../../multi-tenant-analytics/data/SiteConfig";
```

Comment:
@tenebricosa Importing a guide-specific config looks hackish as hell. However, since this is a temporary thing for an experiment, I think we can tolerate that.

Reply:
That was the testing part, I've just forgotten to remove it. Thanx for your attention!